submarine-server/server-api/src/main/java/org/apache/submarine/server/api/spec/LdapSpec.java

```diff
@@ -0,0 +1,49 @@
+/*
+ * Licensed to the Apache Software Foundation (ASF) under one
+ * or more contributor license agreements.  See the NOTICE file
+ * distributed with this work for additional information
+ * regarding copyright ownership.  The ASF licenses this file
+ * to you under the Apache License, Version 2.0 (the
+ * "License"); you may not use this file except in compliance
+ * with the License.  You may obtain a copy of the License at
+ *
+ *   http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing,
+ * software distributed under the License is distributed on an
+ * "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+ * KIND, either express or implied.  See the License for the
+ * specific language governing permissions and limitations
+ * under the License.
+ */
+
+package org.apache.submarine.server.api.spec;
+
+public class LdapSpec {
+
+  /**
+   * Name of the user
+   */
+  private String userName;
+
+  /**
+   * password of the user
+   */
+  private String password;
+
+  public String getUserName(){
+    return userName;
+  }
+
+  public String getPassword(){
+    return password;
+  }
+
+  public void setUserName(String user_name) {
```

Comment:
function setUserName param is `user_name`,  `this.userName = userName;` corrent??